### ISSUE TITLE
Tweaks and vscode specific features

### ISSUE THREAD

**Issue by cometeer** [sallar/vscode-duotone-dark#1]
Hello @sallar,

I've made some small tweaks for a personal version of the theme. (mostly the UNO5 was just to close to the background color for my liking) but i've also added some properties for handling the multi select features the way viscose works with them (I have no clue how text mate works with this as i've never used that)

Are you at all interested in contributions that deviate from the original theme (possibly as alternate version in the same package) or would you prefer I changed the name and make this a new package on the marketplace?

**Comment by sallar** [comment]
Hi @cometeer, 

I'd appreciate the changes. Yes we can put that in the same package if you wouldn't mind. Send a PR and I will merge it. Also if there's vscode-specific changes that you can make to theme.xml file to make it better, that would be welcome too :)

**Comment by sallar** [comment]
Hi @cometeer, I missed your PR 😱 Really sorry. Will check it out ASAP.

**Comment by cometeer** [comment]
Sure thing, i've got my version as seen in the PR running locally so no rush :)

**Comment by sallar** [comment]
@cometeer It would be nice if we could make screenshots of the new changes... but even if we don't i think it's still fine. Or maybe a better name than "Alt"?

**Comment by cometeer** [comment]
Agreed on the name, though im not really inspired atm for something else. 

Im thinking about doing another round of tweaks this weekend maybe, ill see if i can generate pictures after im done with that.
